02-Reproject-Raster-In-R.Rmd

```diff
@@ -53,25 +60,26 @@ RStudio to write your code.
 * **rgdal:** `install.packages("rgdal")`
 
 ####Data to Download
+Download the raster files teaching dataset:
```

Comment:
Download the NEON raster data teaching subset:

Or delete altogether. the link is probably sufficient?